diff -Naur a/mqtt.py b/mqtt.py
--- a/mqtt.py	2018-03-09 17:09:32.000000000 +0100
+++ b/mqtt.py	2018-03-09 17:09:03.000000000 +0100
@@ -19,10 +19,10 @@
     HAVE_TLS = False

 # read initial config files
-dirname = os.path.dirname(os.path.abspath(__file__)) + '/'
+dirname = '/var/lib/broadlink-mqtt/'
 logging.config.fileConfig(dirname + 'logging.conf')
-CONFIG = os.getenv('BROADLINKMQTTCONFIG', dirname + 'mqtt.conf')
-CONFIG_CUSTOM = os.getenv('BROADLINKMQTTCONFIGCUSTOM', dirname + 'custom.conf')
+CONFIG = os.getenv('BROADLINKMQTTCONFIG', '/etc/broadlink-mqtt.conf')
+CONFIG_CUSTOM = os.getenv('BROADLINKMQTTCONFIGCUSTOM', '/etc/broadlink-mqtt-custom.conf')


 class Config(object):

