diff -Naur a/mqtt.py b/mqtt.py
--- a/mqtt.py	2018-03-02 05:18:30.000000000 +0100
+++ b/mqtt.py	2018-03-09 17:05:26.000000000 +0100
@@ -11,7 +11,6 @@
 import sched
 import json
 from threading import Thread
-from test import TestDevice

 HAVE_TLS = True
 try:
@@ -241,8 +240,6 @@
             device = configure_device(device, topic_prefix + mqtt_subprefix)
             devices_dict[mqtt_subprefix] = device
         return devices_dict
-    elif device_type == 'test':
-        return configure_device(TestDevice(cf), topic_prefix)
     else:
         host = (cf.get('device_host'), 80)
         mac = bytearray.fromhex(cf.get('device_mac').replace(':', ' '))

